dao/mongo.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 
 	"github.com/companieshouse/chs.go/log"
-	"github.com/companieshouse/lfp-pay-api-core/models"
+	"github.com/companieshouse/penalty-payment-api-core/models"
```

Comment:
The 'lfp-pay-api-core' repo hasn't been renamed yet.

Reply:
In flight with Dileep.